src/Teacher.js

```diff
@@ -1,146 +1,94 @@
 const util = require("../src/Util.js");
 const database = require("../src/Database.js");
 
-function QuestionToHTMLStrTeacher(assessment, question, index)
```

Comment:
Please do not delete these functions as they are used by other parts of the web page which still need to be visible to the user